modules/n1ql/pages/n1ql-language-reference/orderby.adoc

```diff
@@ -153,33 +153,41 @@ ORDER BY city DESC, name ASC;
 ====
 [source,n1ql]
 ----
-SELECT name
-  FROM `travel-sample`
-  WHERE type = "landmark"
-UNION SELECT name
-  FROM `travel-sample`
-  WHERE type = "hotel"
-ORDER BY name ASC;
+SELECT name AS sort, type
```

Comment:
Ah, so does `type` still exist in the new sample?  

Not sure why the updated query brings in type, and renames name to sort.

Reply:
Aha, nice, you explain the rename just below!

Reply:
Yes, each collection still has the type field.